e2e/multinet_test.go

```diff
@@ -9,72 +9,105 @@ import (
 	"testing"
 
 	"github.com/weaveworks/ignite/e2e/util"
+	api "github.com/weaveworks/ignite/pkg/apis/ignite"
+	meta "github.com/weaveworks/ignite/pkg/apis/meta/v1alpha1"
+	igniteConstants "github.com/weaveworks/ignite/pkg/constants"
+	"github.com/weaveworks/ignite/pkg/dmlegacy"
+	"github.com/weaveworks/ignite/pkg/metadata"
+	"github.com/weaveworks/ignite/pkg/network"
+	"github.com/weaveworks/ignite/pkg/operations"
+	"github.com/weaveworks/ignite/pkg/providers"
+	igniteDocker "github.com/weaveworks/ignite/pkg/providers/docker"
+	"github.com/weaveworks/ignite/pkg/providers/ignite"
+	"github.com/weaveworks/ignite/pkg/runtime"
+	igniteUtil "github.com/weaveworks/ignite/pkg/util"
 	"gotest.tools/assert"
 )
 
-// TestMultipleInterface tests that a VM's can be configured with more than 1 interface
-func TestOneExtraInterface(t *testing.T) {
+var (
+	multinetVM  = "e2e-test-vm-multinet"
+	sanboxImage = "weaveworks/ignite:dev"
+	kernelImage = "weaveworks/ignite-kernel:5.4.108"
+	vmImage     = "weaveworks/ignite-ubuntu"
+)
+
+func startAsyncVM(t *testing.T, intfs []string) (*operations.VMChannels, string) {
+
 	assert.Assert(t, e2eHome != "", "IGNITE_E2E_HOME should be set")
+	igniteUtil.GenericCheckErr(providers.Populate(ignite.Preload))
 
-	vmName := "e2e-test-vm-multinet"
+	_ = igniteDocker.SetDockerRuntime()
+	_ = igniteDocker.SetDockerNetwork()
```

Comment:
It's a natural consequence of globals being side-effected here that these tests cannot be parallelized with other test runs that need different values.
This doesn't affect the existing test suites which make individual invocations of the ignite binary.
Also, we already run our e2e suite in serial anyway.

All of this is fine, but it's worth noting. :+1: